environments/stfc-base/inventory/group_vars/all/variables.yml

```diff
@@ -186,3 +186,8 @@ azimuth_authenticator_federated_enabled: yes
 azimuth_authenticator_federated_provider: "openid"
 azimuth_authenticator_federated_protocol: "openid"
 azimuth_authenticator_federated_label: Login using IAM
+
+#######################################################################
+# Disabling FIP workstations
+azimuth_caas_workstation_ssh_enabled: no
```

Comment:
I'd prefer you put this in environments/stfc-production (so we can keep testing this on staging/dev clusters)